src/components/task/task-cards-list/task-card/task-card.tsx

```diff
@@ -0,0 +1,95 @@
+import React, {useCallback} from "react";
+import {completeTask, Task} from "../../task-slice";
+import CardHeader from "@mui/material/CardHeader";
+import Box from "@mui/material/Box";
+import IconButton from "@mui/material/IconButton";
+import CheckIcon from "@mui/icons-material/Check";
+import Typography from "@mui/material/Typography";
+import Tooltip from "@mui/material/Tooltip";
+import EditIcon from "@mui/icons-material/Edit";
+import CardActions from "@mui/material/CardActions";
+import {ExpandMore} from "../../../styled/expand-more";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import Collapse from "@mui/material/Collapse";
+import CardContent from "@mui/material/CardContent";
+import Card from "@mui/material/Card";
+import {useAppDispatch, useAppSelector} from "../../../../app/hooks";
+import dayjs from "dayjs";
+import {selectValueCalendarDay} from "../../../../features/main/right-section/right-section-slice";
+
+interface TaskCardProps {
+    task: Task;
+}
+
+export const TaskCard = (props: TaskCardProps) => {
+    const dispatch = useAppDispatch();
+    const {task} = props;
+    const [expanded, setExpanded] = React.useState(false);
+    const value = useAppSelector(selectValueCalendarDay);
+
+    const handleExpandTask = useCallback(() => {
+        setExpanded((prev) => !prev);
+    }, []);
+
+    const handleRemoveTask = useCallback((taskId: string) => {
```

Comment:
I think it is no a good naming here =). Do you actually removing task or making it completed?